### PR TITLE
Updating Neural Network and added helper functions

### LINKED ISSUE #5: Feature reconstruction activation function
https://github.com/WayScience/CytoTrajectory/blob/e78794a91f590649f18463ac424a01d1c2624639/notebooks/nbconverted/VAE_test.py#L60

The sigmoid activation funtion can't capture the unbounded features that range outside [0, 1]

### PR DESCRIPTION
**Pull request by axiomcura**
This PR Fixes #5  and #4

### CODE REVIEW

File: cytotraj/utils/io.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    Returns
+    -------
+    pd.DataFrame
+        loaded image-based profile
+    """
+
+    # type checking
+    fpath = _is_valid_path(fpath)
+
+    # loading into pandas dataframe
```
Comment (MattsonCam) [inline cytotraj/utils/io.py:107]: May also consider include tsvs' here

File: cytotraj/utils/io.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        path to dataset
+
+    Returns
+    -------
+    pd.DataFrame
+        loaded image-based profile
+    """
+
+    # type checking
+    fpath = _is_valid_path(fpath)
```
Comment (MattsonCam) [inline cytotraj/utils/io.py:105]: Does this also check if the path is a file?

File: notebooks/nbconverted/VAE_test.py [hunk 3]
```diff
@@ -57,7 +61,7 @@ def __init__(self, input_dim, latent_dim):
             nn.Linear(32, 64),
             nn.ReLU(),
             nn.Linear(64, input_dim),
-            nn.Sigmoid(),
+            nn.ReLU(),
```
Comment (MattsonCam) [inline notebooks/nbconverted/VAE_test.py]: It might be better to exclude this, since ReLU maps to only non-negative values

File: notebooks/nbconverted/VAE_test.py [hunk 4]
```diff
@@ -4,20 +4,24 @@
...
 import pandas as pd
 import torch
 import torch.nn as nn
 import torch.optim as optim
-from cytotraj.utils.model_utils import load_data
 from torch.utils.data import Dataset
 
+warnings.simplefilter("ignore")
+sys.path.append("../")
+from cytotraj.utils.io import load_data  # noqa
```
Comment (MattsonCam) [inline notebooks/nbconverted/VAE_test.py:19]: Does this fix #4?

**Review by MattsonCam — CHANGES_REQUESTED** [review]
I left some comments, and I think there is just one minor modification, otherwise it LGTM. Great job @axiomcura!

File: cytotraj/utils/io.py (hunk 2 above)
Comment (axiomcura) [reply cytotraj/utils/io.py:105]: nope but added!

File: cytotraj/utils/io.py [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+        value = _convert_to_pathlib_object(value)
+        return value
+    if isinstance(value, pathlib.Path):
+        value = value.resolve(strict=True)
+        return value
+
+
+def load_data(fpath: str | pathlib.Path) -> pd.DataFrame:
+    """Loads in image-base profiles as a pandas dataframe
+
```
Comment (axiomcura) [inline cytotraj/utils/io.py:76]: added!

File: notebooks/nbconverted/VAE_test.py (hunk 4 above)
Comment (axiomcura) [reply notebooks/nbconverted/VAE_test.py:19]: yes! the module is created under cytotraj/utils/io.py

**Comment by axiomcura** [comment]
Thanks for all the comments and suggestions @MattsonCam going to merging!
